smart-contracts/contracts-common/concordium-contracts-common/src/types.rs

```diff
@@ -1062,6 +1064,11 @@ impl<'de> SerdeDeserialize<'de> for Duration {
     }
 }
 
+/// Canonical address of an account, as raw bytes.
```

Comment:
We need more documentation here, Canonical address is not self-explanatory.
It also needs to mention the account alias and when this is type interesting to use.